docs/configuration.md

```diff
@@ -678,6 +678,14 @@ responders:
 # Priority level of alert. Possible values are P1, P2, P3, P4, and P5.
 [ priority: <tmpl_string> ]
 
+# Whether or not to send a request to update alert message every time every time an alert is sent to OpsGenie
+# By default, the message of the alert is never updated in OpsGenie, the new message only appears in activity log
+[ update_message: <boolean> | default = false ]
+
+# Whether or not to send a request to update alert description every time every time an alert is sent to OpsGenie
+# By default, the description of the alert is never updated in OpsGenie
+[ update_description: <boolean> | default = false ]
```

Comment:
Can we have maybe a single boolean for both?

Reply:
we could, I just don't know if everyone uses OpsGenie the same way, maybe not everybody cares about both message and description

Reply:
I personally agree with @roidelapluie, the functionnal need is to update or not the alert, not each field independently. 
Example If we want to update a new field in the future (maybe the severity idk), we don't want to add a third configuration option.

Reply:
I agree in principle but unfortunately the OpsGenie api needs a separate request for each one, if somebody is hitting api rate limits, they might want to only update some fields, updating all of them on every alert means 3x the request volume.

Reply:
@roidelapluie @alistarle If you think a single boolean is the way to go even when a request is needed to update each field, I don't really have a problem with it. Let me know so that I can update the PR

Reply:
Indeed I understand this point, for me you can leave this as is, if it can make alertmanager more accessible for small business because it can save opsgenie API call

Reply:
Finally I think it is better to stick to the functionnal need, which is to just update the alert, because it is something who will never change, and we should avoid configuration change in the future.

We cannot know what will happen in the future regarding opsgenie API to allow bulk update, billing plan to allow more API call, customer who will upgrade to an higher plan... We will be able to update the code, but if it require config change it would be a pain for users.

Reply:
ok, I will update the PR to only use one option for both, do I call it `update_alerts` or just `update`?

Reply:
changed to a single `update_alerts` option